.github/workflows/release.yaml

```diff
@@ -2,26 +2,52 @@ name: Release Charts
 
 on:
   push:
-    branches:
-      - main
+    tags:
+      - "v*.*.*"
```

Comment:
How will this tag be created? Person releasing self-hosted release manually creates a release tag?

Reply:
The idea, at least to start, would be to manually tag. When new tag is created that would kick off the release workflow

Reply:
During this initial stage, we'll have two step helm-chart release process:

1. Create PR -> Update the app version in Chart.yaml -> Commit PR to main
2. Create release tag, the tag is at copia-0.21.0, so the next one will be copia-0.22.0

Should this tag version match to app version or should it have its own separate version for help chart? In the past we have had releases where helm version alone changed without change to appversion.

Reply:
1. Merge to main with whatever changes are required (including app version)
2. git tag would then refer to a commit on the main branch and map to the helm chart version, so it would be `v1.2.3` of the helm chart could refer to any arbitrary version of the app.